client/packages/common/src/api/GqlContext.tsx

```diff
@@ -55,10 +55,25 @@ const handleResponseError = (errors: ResponseError[]) => {
   throw new Error(details || error?.message || 'Unknown error');
 };
 
+const ignoredQueries = ['refreshToken', 'syncInfo'];
+
+const shouldIgnoreQuery = (definitionNode: DefinitionNode) => {
+  const operationNode = definitionNode as OperationDefinitionNode;
+  if (!operationNode) return false;
```

Comment:
ah, yes  that line is due to a misunderstanding of type assertion. I'll just take it out, the following line will pick up an incorrect object.

( was put in because the `DefinitionNode` could be any of these types and I wanted to check that we had the correct type.. but the assertion does not convert and return a null if the conversion fails, it is only asserting that the object should be of a certain type

- DirectiveDefinitionNode
- FragmentDefinitionNode
- OperationDefinitionNode
- SchemaDefinitionNode
- SchemaExtensionNode
- TypeDefinitionNode
- TypeExtensionNode
- TypeSystemExtensionNode
)